<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XViewerClass</class>
 <widget class="QWidget" name="XViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XViewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xviewer.qrc">
    <normaloff>:/XViewer/img/xv.ico</normaloff>:/XViewer/img/xv.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 按钮样式*/
QPushButton:hover
{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 102, 184, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39, 39, 39, 255), stop:1 rgba(0, 102, 184, 255));
border:none;
border-radius:5px;
color:rgb(255, 255, 255);
font:75 12pt &quot;黑体&quot;;
}

QPushButton:!hover
{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 50, 150, 255), stop:0.495 rgba(0, 102, 184, 255), stop:0.505 rgba(0, 102, 184, 255), stop:1 rgba(39, 39, 39, 255) );
border:none;
border-radius:8px;
color:rgb(255, 255, 255);
font:75 12pt &quot;黑体&quot;;
}

/*状态*/
#status
{
color:rgb(255, 255, 255);
font:75 13pt &quot;Adobe Arabic&quot;;
} 


/*顶部样式*/
#head
{
 background-color:#3c3c3a;
}
#logo
{
	background-image: url(:/XViewer/img/logo_150_40.png);
}
#close
{
	background-image: url(:/XViewer/img/close.png);
}
#min
{
	background-image: url(:/XViewer/img/min.png);
}
#max
{
	background-image: url(:/XViewer/img/max.png);
}

#normal
{
	background-image: url(:/XViewer/img/normal.png);
}

/* body 演示 */
#body
{
  background-color:#212121 ;
}
#left
{
  background-color:#252525;
}
#cams
{
background-color:#1e1e1e;
}

/* tab 菜单按钮切换样式*/
#preview
{
background-color:rgb(50, 50, 50);
} 
#playback
{
background-color:rgb(50, 50, 50);
} 
#push_stream
{
background-color:rgb(50, 50, 50);
} 

#preview:checked
{
background-color:rgb(0, 102, 184);
} 
#playback:checked
{
background-color:rgb(0, 102, 184);
}
#push_stream:checked
{
background-color:rgb(0, 102, 184);
}</string>
  </property>
  <widget class="QWidget" name="head" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>50</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="head_button" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QPushButton" name="min">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="close">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="max">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="normal">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>min</zorder>
    <zorder>close</zorder>
    <zorder>normal</zorder>
    <zorder>max</zorder>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>监控中......</string>
    </property>
   </widget>
   <widget class="QPushButton" name="preview">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>预览</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="playback">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>回放</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="push_stream">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>推流</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>软解码</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>硬解+软渲</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>硬解码</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QWidget" name="body" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>801</width>
     <height>611</height>
    </rect>
   </property>
   <widget class="QWidget" name="cams" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>561</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="left" native="true">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>10</y>
      <width>200</width>
      <height>600</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <widget class="QListWidget" name="cam_list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>49</y>
       <width>200</width>
       <height>951</height>
      </rect>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>camera_test</string>
      </property>
      <property name="icon">
       <iconset resource="xviewer.qrc">
        <normaloff>:/XViewer/img/cam.png</normaloff>:/XViewer/img/cam.png</iconset>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="add_cam">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>新增</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_cam_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>5</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_cam_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>5</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="playback_wid" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>571</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QListWidget" name="time_list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>150</width>
       <height>601</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>20:30:59</string>
      </property>
     </item>
    </widget>
    <widget class="XCalendar" name="cal">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>401</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="XPushStreamWidget" name="push_wid" native="true">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>411</width>
      <height>591</height>
     </rect>
    </property>
    <widget class="QLabel" name="push_status">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>451</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 36pt &quot;Adobe 宋体 Std L&quot;;</string>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>拖到此处进行推流</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_url">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="push_lable">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>25</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>RTMP地址：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="filter_type">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>原始</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>磨皮</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>美白</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>边缘</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XCalendar</class>
   <extends>QCalendarWidget</extends>
   <header>xcalendar.h</header>
  </customwidget>
  <customwidget>
   <class>XPushStreamWidget</class>
   <extends>QWidget</extends>
   <header>xpush_stream_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="xviewer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>782</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>1066</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>692</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>max</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>MaxWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>948</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>normal</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>NormalWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_cam</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>AddCam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_cam_2</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>SetCam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_cam_3</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>DelCam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preview</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>Preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playback</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>Playback()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cam_list</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>XViewerClass</receiver>
   <slot>SelectCamera(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cal</sender>
   <signal>clicked(QDate)</signal>
   <receiver>XViewerClass</receiver>
   <slot>SelectDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>time_list</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>XViewerClass</receiver>
   <slot>PlayVideo(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>push_stream</sender>
   <signal>clicked()</signal>
   <receiver>XViewerClass</receiver>
   <slot>PlayPush()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>-103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_url</sender>
   <signal>returnPressed()</signal>
   <receiver>XViewerClass</receiver>
   <slot>SetRtmpurl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>-26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>XViewerClass</receiver>
   <slot>SetDecodeType(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>-80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filter_type</sender>
   <signal>activated(QString)</signal>
   <receiver>XViewerClass</receiver>
   <slot>SetFilterType(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>749</x>
     <y>-80</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>MaxWindow()</slot>
  <slot>NormalWindow()</slot>
  <slot>AddCam()</slot>
  <slot>SetCam()</slot>
  <slot>DelCam()</slot>
  <slot>Preview()</slot>
  <slot>Playback()</slot>
  <slot>SelectCamera(QModelIndex)</slot>
  <slot>SelectDate(QDate)</slot>
  <slot>PlayVideo(QModelIndex)</slot>
  <slot>PlayPush()</slot>
  <slot>SetRtmpurl()</slot>
  <slot>SetDecodeType(QString)</slot>
  <slot>SetDecodeType(int)</slot>
  <slot>SetFilterType(QString)</slot>
 </slots>
</ui>
